qa/multi-cluster-search/src/test/resources/rest-api-spec/test/multi_cluster/30_field_caps.yml

```diff
@@ -73,6 +73,21 @@
   - match: {fields.number.double.searchable:            true}
   - match: {fields.number.double.aggregatable:          true}
 
+  # make sure runtime_mappings section gets propagated
+  - do:
+      field_caps:
+        index: 'my_remote_cluster:field_caps_index_1'
+        fields: [number]
+        body:
+          runtime_mappings:
+            number:
+              type: keyword
+              script:
+                source: "emit(doc['number'].value)"
```

Comment:
I am wondering how this is not throwing an error for trying to access doc_values of the field that is being defined. Would it make sense to use a different field name? You can potentially even skip the script.

Reply:
Though I see that this is also testing shadowing the existing mapped field. Maybe that is not required though as it's besides the scope of this test. We could define a new field as part of runtime_mappings and check that field_caps returns it

Reply:
Done (skipped the `script` part).

I think the script is just not executed during `_field_caps` so it's cleaner not to provide it in the first place.

Reply:
ah right that's it and that is why it was not causing an error :)